From: Jeremy Bicha <jeremy.bicha@canonical.com>
Date: Mon, 28 Mar 2022 11:10:48 -0400
Subject: meson: don't update icon cache

Works around https://bugs.debian.org/1008189

The icon cache doesn't really need to be updated like this on Debian
since we have hooks that do that for us
---
 meson.build                            | 2 +-
 subprojects/extensions-app/meson.build | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/meson.build b/meson.build
index 8d7478f..9df3ead 100644
--- a/meson.build
+++ b/meson.build
@@ -297,7 +297,7 @@ endif
 
 gnome.post_install(
   glib_compile_schemas: true,
-  gtk_update_icon_cache: true
+  gtk_update_icon_cache: false
 )
 
 meson.add_dist_script('meson/generate-manpages.py')
diff --git a/subprojects/extensions-app/meson.build b/subprojects/extensions-app/meson.build
index c260d7a..69daa27 100644
--- a/subprojects/extensions-app/meson.build
+++ b/subprojects/extensions-app/meson.build
@@ -61,7 +61,7 @@ if not meson.is_subproject()
   subdir('po')
 
   gnome.post_install(
-    gtk_update_icon_cache: true
+    gtk_update_icon_cache: false
   )
 
   if appstream_util.found()
